devops/steps/linux/anybuild.yml

```diff
@@ -34,22 +34,24 @@ steps:
 
     readonly abClientExe="$HOME/.local/share/Microsoft/AnyBuild/AnyBuild.sh"
 
-    env -i                                                                \
-      SECRET=$(LLVMPrincipalPassword)                                     \
-      HOME="$HOME"                                                        \
-      PATH="$PATH"                                                        \
-    "$abClientExe"                                                        \
-      --RemoteExecServiceUri $(AnyBuildEnvironmentUri)                    \
-      --NoCheckForUpdates                                                 \
-      --WaitForAgentForever                                               \
-      --DoNotUseMachineUtilizationForScheduling                           \
-      --ClientApplicationId $(LLVMPrincipalAppId)                         \
-      --ClientSecretEnvironmentVariable SECRET                            \
-      --ExperimentName "${{ parameters.BuildName }}"                      \
-      --WhyCacheMiss                                                      \
-      --WhyCacheMissOptions CacheDataStoreKey=${{ parameters.BuildName }} \
-      ${{ parameters.ExtraAnyBuildArgs }}                                 \
-      --                                                                  \
+    env -i                                                                  \
+      SECRET=$(LLVMPrincipalPassword)                                       \
+      HOME="$HOME"                                                          \
+      PATH="$PATH"                                                          \
+    "$abClientExe"                                                          \
+      --RemoteExecServiceUri $(AnyBuildEnvironmentUri)                      \
+      --NoCheckForUpdates                                                   \
+      --WaitForAgentForever                                                 \
+      --DoNotUseMachineUtilizationForScheduling                             \
+      --ClientApplicationId $(LLVMPrincipalAppId)                           \
+      --ClientSecretEnvironmentVariable SECRET                              \
+      --ExperimentName "${{ parameters.BuildName }}"                        \
+      --LogDir .                                                            \
+      --NoSessionLogDir                                                     \
+      --WhyCacheMiss                                                        \
+      --WhyCacheMissOptions CacheDataStoreKey="${{ parameters.BuildName }}" \
```

Comment:
The clang build passes a value for `parameters.BuildName` that contains commas, so if this is not quoted the commas break the command line